arangod/MMFiles/MMFilesCompactorThread.cpp

```diff
@@ -318,13 +318,12 @@ MMFilesCompactorThread::CompactionInitialContext MMFilesCompactorThread::getComp
       return true;
     };
 
-
     bool ok;
     {
       auto physical = static_cast<MMFilesCollection*>(context._collection->getPhysical());
       TRI_ASSERT(physical != nullptr);
       bool const useDeadlockDetector = false;
-      int res = physical->lockRead(useDeadlockDetector, 86400.0);
+      int res = physical->lockRead(trx->tid(), useDeadlockDetector, 86400.0);
```

Comment:
is that the correct parameter order here? isn't `tid` supposed to be the second parameter?